### ISSUE TITLE
download manager leaves tab open

### ISSUE THREAD

**Issue by hhgoten** [QupZilla/qupzilla#2572]
When I open a pdf document , there is a window that asks me wether to save or open the document.
If i say : open , in my case (xubuntu ) it opens fine in qpdfview.
But when I close the pdf-viewer, I am left with a open, empty browser-tab.
I marked "close download-manager after success" , but have not find a way around, that this tab is closed.
If I open some pdf documents for e.g. from my bank, I have a tab for every document.
Please give some advice to configure .
yours sincerely waldi

**Comment by nowrep** [comment]
It works on some pages, and on some it doesn't. Can you please post some url where it can be reproduced?

**Comment by Diablopc** [comment]
e.g. here 
go to https://yandex.fr/search/?text=rtl8723ae+pdf&lr=143
then click on any pdf icon ![](http://i.piccy.info/i9/06dc64612e16acf5820b972c11d7f9e5/1516981993/5157/1199641/Screenshot_20180126_175258.png)

**Comment by Diablopc** [comment]
btw about download manager:
Here is always "Remaining few seconds"
![](http://i.piccy.info/i9/16b8951cb4635b7d5063c8ea248b1fbb/1516982317/41585/1199641/Screenshot_20180126_175726.png)

**Comment by hhgoten** [comment]
@nowrep 
You are right, works on most sites. I did not realize that some sites work, because my problem was nasty, when I go to the mailbox of comdirect (my bank account) .
At  the moment, I don't know, how to give you a link to the site ( my account would be nice ;-))    )
Maybe you can do with your own account, if you got one . Another bank (Targo) works fine. Maybe its just the comdirect. 
How can I give some hints ? code of the page ?

**Comment by nowrep** [comment]
@Diablopc On yandex it doesn't work because it redirects to different host and the logic that is used to detect which tab to close (because there is no API in QtWebEngine to get the page that initiated the download) just fails in that case.
